ENGAGEHF/Bluetooth/Views/MeasurementLayer.swift

```diff
@@ -0,0 +1,49 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+private struct PreviewWrapperMeasurementLayer: View {
```

Comment:
I would suggest that we keep the main views up at the top of the view and add the preview wrapper at the bottom close to the previews.

I would suggest to switch to a following setup that defines the wrapper types right in the preview macros:
```swift
#Preview {
    struct PreviewWrapperMeasurementLayer: View {
        @Environment(MeasurementManager.self) private var measurementManager
        
        
        var body: some View {
            MeasurementLayer()
                .onAppear {
                    measurementManager.loadMockMeasurement()
                }
        }
    }
    
    return PreviewWrapperMeasurementLayer()
        .previewWith(standard: ENGAGEHFStandard()) {
            MeasurementManager()
        }
}
```